src/functions.php

```diff
@@ -420,3 +420,37 @@ function env($name = null, $value = null)
         return null;
     }
 }
+
+/**
+ * Adds a global argument
+ *
+ * @param string $name
+ * @param array  $argument
+ *
+ * @return mixed
+ */
+function argument($name, array $argument = null)
+{
+    if (null === $argument) {
+        return Deployer::get()->getInput()->getArgument($name);
```

Comment:
When doing this I have to inform every `TaskCommand` about the created arguments/options in order to have them available in that specific context.